### PR TITLE
COMMERCE-10007 Automate tests for user story COMMERCE-8133 (2/2)

### PR DESCRIPTION
**Pull request by 22LucaGiannotta22**

### CODE REVIEW

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce

File: portal-web/test/functional/com/liferay/portalweb/macros/Permissions.macro [hunk 1]
```diff
@@ -63,6 +63,15 @@ definition {
...
 
+	macro addPermissionViaAPI {
+		JSONPermission.addPermission(
+			actionId = "${resourceAction}",
+			groupName = "${groupName}",
+			name = "${resourceName}",
+			roleTitle = "${roleTitle}",
+			roleType = "${roleType}");
+	}
+
```
Comment (lorenzolapescara) [inline portal-web/test/functional/com/liferay/portalweb/macros/Permissions.macro:74]: To assign permission to a role via API there is already `Permissions.definePermissionViaJSONAPI()`.
Please use that one and remove this macro. Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAddressPermissions.testcase [hunk 2]
```diff
@@ -50,6 +50,78 @@ definition {
...
+				userEmailAddress = "test@liferay.com");
+		}
+
+		task ("And adds the view addresses permission to the buyer role") {
+			Permissions.addPermissionViaAPI(
+				groupName = "Minium",
+				resourceAction = "VIEW_ADDRESSES",
+				resourceName = "com.liferay.account.model.AccountEntry",
+				roleTitle = "Buyer",
+				roleType = "account");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAddressPermissions.testcase:101]: See the comment below.

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce

**Pull request closed without merging**